adr/0021-sre-cloud-support.md

```diff
@@ -0,0 +1,80 @@
+# Provide a service catalog
+
+* Status: accepted
+* Deciders: durandom
+* Date: 2022-07-25
+
+Technical Story: https://github.com/open-services-group/scrum/issues/4
+
+## Context and Problem Statement
+
+In general, the individual or team that onboards a service often assumes the sole role of operating that service
+indefinitely. There are various issues associated with this:
+
+* Other members of the SRE team are unable to offer sufficient support for the service, support here can refer to:
+  * Operational support (improvements, incidents, fixes, patches, upgrades, etc.)
+  * Responding to end user questions via GitHub, Slack, Mailing List pertaining to the service
+* Siloed knowledge results in fewer team members being able to perform PR reviews pertaining to the service
+* Documentation for the app/service suffers, since now only one person is able to write/understand them
+* The one (or few) engineers responsible for maintaining the service have to devote large portions of their engineer
```

Comment:
```suggestion
* The one (or few) engineers responsible for maintaining the service have to devote large portions of their engineer time to operate the service, as opposed to building out new features in Operate First
```

Reply:
removing also the period "." to align the layout (some list items have periods, some don't)